src/main/groovy/org/shipkit/internal/gradle/GitPlugin.java

```diff
@@ -109,11 +113,25 @@ public void execute(final Task t) {
             }
         });
 
-        TaskMaker.execTask(project, COMMIT_CLEANUP_TASK, new Action<Exec>() {
+        TaskMaker.task(project, PERFORM_GIT_COMMIT_CLEANUP_TASK, Task.class, new Action<Task>() {
+            public void execute(final Task t) {
+                t.setDescription("Performs " + SOFT_RESET_COMMIT_TASK + " and " + GIT_STASH_TASK + " tasks.");
+                t.dependsOn(SOFT_RESET_COMMIT_TASK, GIT_STASH_TASK);
+            }
+        });
+
+        TaskMaker.execTask(project, GIT_STASH_TASK, new Action<Exec>() {
+            public void execute(final Exec t) {
+                t.setDescription("Stashes current changes");
+                t.commandLine("git", "stash");
```

Comment:
I'd prefer the version without `save` as 3rd argument (since git stash without any arguments is equivalent to git stash save).